deploy-cloud-functions/src/cloudFunction.ts

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { cloudfunctions_v1 } from 'googleapis';
+
+export type EnvVar = {
+  [key: string]: string;
```

Comment:
It just denotes a k/v structure where key is a string and value is a string. I defined it this way instead of 
```ts
export type EnvVar = {
  name: string;
  value: string;
};
``` 
because CF expects it as a map of k/v.